scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml

```diff
@@ -11,7 +11,6 @@ clang_binary_name: "clang-8"
 
 apt_packages:
   - "curl"
-  - "clang-format-7"
```

Comment:
@anakrish suggested we keep clang-format-7. I'll let him comment on the reason, but I believe you may want to revert line 14 here.